portal/src/graphql/portal/schema.graphql

```diff
@@ -30,6 +30,9 @@ type App implements Node {
   """The ID of an object"""
   id: ID!
 
+  """"""
+  planName: AppConfig!
```

Comment:
The type should be string. After we correct the type in the code, re-generate the schema should correct this as well.